_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -25,32 +25,34 @@ The rule applies to any [keyboard event][]:
 
 ## Expectation 1
 
-For each test target:
- - an [instrument][] to prevent any result of the [keyboard event][] is available; or
- - if the [event target][] doesn't have [focus][], any result of the event is prevented; or
- - an [instrument][] is available to prevent any result of the [keyboard event][] when the [keyboard event][] method `getModifierState` returns `false`.
+For each test target at least one of the following is true:
+ - (**Disable**:) a set of one or more [instruments][instrument] are available to prevent any result of the [keyboard event][] within a 10 minute time span of the [event firing][]; or
+ - (**Focus**:) any result of the event is prevented if the [event target][] doesn't have [focus][] within a 10 minute time span of the [event firing][]; or
+ - (**Remap**:) a set of one or more [instruments][instrument] are available to prevent any result of the [keyboard event][] when the [keyboard event][] method `getModifierState` returns `false` within a 10 minute time span of the [event firing][].
```

Comment:
The "within 10 minutes" part need to be reformulate to make clear what (and how) it applies to.

"prevent any result if the target doesn't have focus within 10 minutes" reads more like "prevent any result (if the target doesn't have focus within 10 minutes)" which makes little sense (what does it means to "not have focus within 10 minutes"?)